main_test.py

```diff
@@ -5,58 +5,226 @@
 
 from PIL import Image
 
-from main import app
+from app import app, init_app
+
+TEST_SCHEDULES = {
+    "aaardvark": {
+        "classes": [
+            [
+                { "period": "A", "room": "TALI-206", "name": "Advanced Spanish: Literature", "teacher_username": "kviolette", "department": "Spanish"},
+                { "period": "B","room": "TMAC-300", "name": "PE: Customized Training to Maximize Performance (US)", "teacher_username": "mhayes", "department": "Physical Education" },

```

Comment:
Nit: It might be good to "sub" out the course names for something more reasonable - i.e. replace `P.E. Customized Training to Maximize Performance` with `PE`, like we did for `aaardvark`. This is more of a style thing though, so there's also an argument for keeping it as is.